File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int num1 = scan.nextInt();
        int num2 = scan.nextInt();
        int num3 = scan.nextInt();

        if (num1 == num2 && num2 == num3)
            System.out.println(10000 + num1 * 1000);
        else if (num1 == num2 || num1 == num3)
            System.out.println(1000 + num1 * 100);
        else if (num2 == num3)
            System.out.println(1000 + num2 * 100);
        else
            System.out.println(Math.max(Math.max(num1, num2), num3) * 100);
    }
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.c
#include <stdio.h>

int main()
{
    int num1, num2, num3;
    scanf("%d %d %d", &num1, &num2, &num3);

    if (num1 == num2 && num2 == num3)
        printf("%d\n", 10000 + num1 * 1000);
    else if (num1 == num2 || num1 == num3)
        printf("%d\n", 1000 + num1 * 100);
    else if (num2 == num3)
        printf("%d\n", 1000 + num2 * 100);
    else
    {
        int max = num1;
        if (max < num2)
            max = num2;
        if (max < num3)
            max = num3;
        printf("%d\n", max * 100);
    }

    return 0;
}